packages/web-app-files/src/composables/actions/files/useFileActionsCreateNewFolder.ts

```diff
@@ -115,13 +115,13 @@ export const useFileActionsCreateNewFolder = ({
     return [
       {
         name: 'create-folder',
-        icon: 'add',
+        icon: 'folder',
         handler,
         label: () => {
-          return $gettext('Create new Folder')
+          return $gettext('New Folder')
         },
-        isEnabled: ({ resources }) => {
-          return true
+        isEnabled: () => {
+          return unref(currentFolder)?.canUpload({ user: store.getters.user })
```

Comment:
this should use `canCreate` instead of  `canUpload`. Weirdly both `canUpload` and  `canCreate` are implemented in the same way (they check if the dav permission for folder creation is present), so you could argument that it doesn't make a difference in functionality... but semantically the creation check is the right one for folder creation. 

Side note: For the other composable (creating files) I prefer the `canUpload` check (you already have it) because if one can upload a file that's the same as creating a file. So that one should stay like it is. 👍